app/server.js

```diff
@@ -5,6 +5,18 @@ const messageService = require('./services/message-service')
 
 async function createServer () {
   // Create the hapi server
+
+  const catboxOptions = {
+    host: config.redisHost,
+    port: config.redisPort,
+    password: config.redisPassword,
+    partition: config.redisPartition
+  }
+
+  if (config.isProd) {
```

Comment:
this could also be done inline in options via a ternary, which would make it clear that tls is `undefined` for development.

I'd be tempted to push the catboxOptions down to the config.js  to keep config logic out of the server.js

Reply:
Agreed and done.